src/http/mime_type.zig

```diff
@@ -89,28 +81,19 @@ pub const Category = enum {
     }
 };
 
-pub const none = MimeType.initComptime("", .none);
-pub const other = MimeType.initComptime("application/octet-stream", .other);
-pub const css = MimeType.initComptime("text/css;charset=utf-8", .css);
-pub const javascript = MimeType.initComptime("text/javascript;charset=utf-8", .javascript);
-pub const ico = MimeType.initComptime("image/vnd.microsoft.icon", .image);
-pub const html = MimeType.initComptime("text/html;charset=utf-8", .html);
-// we transpile json to javascript so that it is importable without import assertions.
-pub const json = MimeType.initComptime("application/json;charset=utf-8", .json);
+pub const none = MimeType{ .value = "", .category = .none };
+pub const other = MimeType{ .value = "application/octet-stream", .category = .other };
+pub const css = MimeType{ .value = "text/css;charset=utf-8", .category = .css };
+pub const javascript = MimeType{ .value = "text/javascript;charset=utf-8", .category = .javascript };
+pub const ico = MimeType{ .value = "image/vnd.microsoft.icon", .category = .image };
+pub const html = MimeType{ .value = "text/html;charset=utf-8", .category = .html };
+pub const json = MimeType{ .value = "application/json;charset=utf-8", .category = .json };
 pub const transpiled_json = javascript;
-pub const text = MimeType.initComptime("text/plain;charset=utf-8", .html);
-pub const wasm = MimeType.initComptime(
-    "application/wasm",
-    .wasm,
-);
-fn initComptime(comptime str: string, t: Category) MimeType {
-    return MimeType{
-        .value = str,
-        .category = t,
-    };
-}
+pub const text = MimeType{ .value = "text/plain;charset=utf-8", .category = .html }; // why??
+pub const wasm = MimeType{ .value = "application/wasm", .category = .wasm };
+pub const form_urlencoded = MimeType{ .value = "application/x-www-form-urlencoded", .category = .application };
```

Comment:
Changing the encoding is an unnecessary breaking change 
```suggestion
pub const form_urlencoded = MimeType{ .value = "application/x-www-form-urlencoded;charset=utf-8", .category = .application };
```

Reply:
Good catch, thank you